lib/mongo/crypt/context.rb

```diff
@@ -106,6 +114,11 @@ def raise_from_status
         status.raise_crypt_error
       end
 
+      # Indicate that state machine is done feeding I/O responses back to libmongocrypt
+      def mongo_done
```

Comment:
`mongo_xxx` method names to me imply that they are interacting with the server. If they are interacting with mongocryptd/libmongocryptd can the prefix be changed to `mongocrypt`, `lmc`, `mc`, etc.?